### PR TITLE
Add Href to found-a-bug

### PR DESCRIPTION
**Pull request by wolflord212312**
Reason so people can click github page and get directed to keettingpowered github and from there go to create a ISSUE

### CODE REVIEW

**Comment by wolflord212312** [comment]
Did NOT Link to a specific repo as one can not be sure if its a launcher issue or a game issue 🤷 
So Deal with it or fight over it
I help?

**Comment by wolflord212312** [comment]
Will say red lost me about this "safe answer []()"

then said there should be safe somewhere in this file that was used, Cant Find what he is talking about in this file 

I am unsure what he is saying, Gonna leave it as is

File: src/data/faq/questions.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const questionsAndAnswers = [
     {
         id: "found-a-bug",
         question: "I found a bug, what do I do?",
-        answer: "Go to our github page and create an issue in the correct repository."
+        answer: "Go to our <a href='${global.githubUrl}'>github page</a> and create an issue in the correct repository."
```
Comment (C0D3-M4513R) [inline src/data/faq/questions.js:28]: There is no safe answer

**Review by C0D3-M4513R — CHANGES_REQUESTED** [review]

File: src/data/faq/questions.js (hunk 1 above)
Comment (C0D3-M4513R) [reply src/data/faq/questions.js:28]: Also please redirect to Ketting here. 99% of the issues belong there.
For that 1%, we can always move them to the correct repo.
Use `global.softwareUrl()` for that.

**Comment by C0D3-M4513R** [comment]
Superseded by: #6

**Comment by C0D3-M4513R** [comment]
nvm. We apparently cannot use global or app in that context.

**Pull request closed without merging**